app/src/main/java/com/alexandruvalentinconstantin/newsreader/BeginTransitionToFragmentActivity.java

```diff
@@ -1,20 +1,21 @@
 package com.alexandruvalentinconstantin.newsreader;
 
-import androidx.appcompat.app.AppCompatActivity;
-
 import android.os.Bundle;
 
-import com.alexandruvalentinconstantin.newsreader.ui.main.MainFragment;
+import androidx.appcompat.app.AppCompatActivity;
 
-public class MainActivity extends AppCompatActivity {
+import com.alexandruvalentinconstantin.newsreader.feature.newslist.fragment.NewsListFragment;
+
+public class BeginTransitionToFragmentActivity extends AppCompatActivity {
```

Comment:
Let's name this activity oriented towards the feature, not to the implementation details, such as if we decide to change what we do inside such class, we should not need to change the name of the activity aswell. Let's keep it simple and less original. Maybe something like NewsActivity.